File: KuantumKaosYonetimi/Python/KuantumKaosYonetimi/anti_madde.py
# anti_madde.py
from .kuantum_nesnesi import KuantumNesnesi
from .i_kritik import IKritik

# C. Nesne Çeşitleri: AntiMadde (IKritik'i uygular)
class AntiMadde(KuantumNesnesi, IKritik):
    
    def __init__(self, id_str: str, stabilite: float):
        # Tehlike seviyesi: 10
        super().__init__(id_str, 10, stabilite) 

    # AnalizEt() (Çok Biçimlilik/Polymorphism)
    def AnalizEt(self):
        # Stabilite 25 birim düşer.
        self.Stabilite -= 25
        print("Evrenin dokusu titriyor...") # Ek uyarı
        print(f"[ID: {self.ID}] Anti Madde analizi yapıldı. Stabilite -25. Yeni Stabilite: {self.Stabilite:.2f}%")

    # AcilDurumSogutmasi() (IKritik Uygulaması)
    def AcilDurumSogutmasi(self):
        # Stabiliteyi +50 artırır (Setter max 100'ü halleder).
        self.Stabilite += 50
        print(f"[ID: {self.ID}] Anti Madde soğutuldu! Stabilite +50. Yeni Stabilite: {self.Stabilite:.2f}%")

File: KuantumKaosYonetimi/Javascript/KuantumKaosYonetimi/AntiMadde.js
// AntiMadde.js
const KuantumNesnesi = require('./KuantumNesnesi');

class AntiMadde extends KuantumNesnesi {
    constructor(id, stabilite) {
        super(id, 10, stabilite); // TehlikeSeviyesi: 10
    }

    // AnalizEt() (Çok Biçimlilik/Polymorphism)
    AnalizEt() {
        this.Stabilite -= 25;
        console.log("Evrenin dokusu titriyor..."); // Ek uyarı
        console.log(`[ID: ${this.ID}] Anti Madde analizi yapıldı. Stabilite -25. Yeni Stabilite: ${this.Stabilite.toFixed(2)}%`);
    }

    // AcilDurumSogutmasi() (IKritik Uygulaması)
    AcilDurumSogutmasi() {
        this.Stabilite += 50; // Setter max 100'ü halleder.
        console.log(`[ID: ${this.ID}] Anti Madde soğutuldu! Stabilite +50. Yeni Stabilite: ${this.Stabilite.toFixed(2)}%`);
    }
}

module.exports = AntiMadde;